lighting_import_attachment_product/models/lighting_import_attachment_file_product.py

```diff
@@ -0,0 +1,12 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class LightingImportAttachmentFileProduct(models.Model):
+    _name = "lighting.import.attachment.file.product"
+    _description = "Lighting Import Attachment File Product"
+
+    product_id = fields.Many2one(comodel_name="product.product")
+    file_id = fields.Many2one(comodel_name="lighting.import.attachment.file")
```

Comment:
Define `required`, `tracking`, `cascade`, etc... and sql constraints (if any) at the very beginning of coding, if you don't there's a very high chances you forget.